website/static/js/fangorn.js

```diff
@@ -223,6 +223,9 @@ function resolveIconView(item) {
         return returnView('link');
     }
     if (item.data.nodeType === 'project') {
+        if (item.data.parentIsFolder && item.data.isFolder && item.data.isPointer) {
+            return returnView('collection', item.data.category);
```

Comment:
what about 
returnView('collection')
?
the second parameter is not necessary in this case.